components/clientComponents/forms/Review/Review.tsx

```diff
@@ -22,9 +22,69 @@ type QuestionAnswer = {
   [x: string]: string;
 };
 
+const QuestionsAnswers = ({
+  group,
+  formRecord,
+  lang,
+}: {
+  group: ReviewGroup;
+  formRecord: TypeOmit<FormRecord, "name" | "deliveryOption">;
+  lang: string;
+}): React.ReactElement => {
+  const { t } = useTranslation("review");
+  function getElementNameById(id: string | number) {
+    const element = formRecord.form.elements.find((item) => String(item.id) === String(id));
+    return element ? element.properties?.[getLocalizedProperty("title", lang)] : t("unknown");
+  }
+  function getQuestionAndAnswer(element: QuestionAnswer) {
+    const question = getElementNameById(Object.keys(element)[0]) as string;
+    const answer = Object.values(element)[0] as string;
+    return { question, answer };
+  }
+  return (
+    <dl className="my-10">
+      {Array.isArray(group.elements) &&
+        group.elements.map((element) => {
+          const { question, answer } = getQuestionAndAnswer(element);
+          return (
+            <div key={Object.keys(element)[0]} className="mb-8">
+              <dt className="font-bold mb-2">{question}</dt>
```

Comment:
Do you have the Tailwind ext installed ?

Will catch things like these classes being in the wrong order --- if not let's chat later about this.

<img width="400" alt="Screenshot 2024-07-18 at 7 18 17 AM" src="https://github.com/user-attachments/assets/e5288a50-ab91-4ce9-883e-e35dae8ca46e">


<img width="400" alt="Screenshot 2024-07-18 at 7 18 49 AM" src="https://github.com/user-attachments/assets/22706973-daf3-4e91-8ef4-a7cb67d0a36d">